components/admin_console/admin_navbar_dropdown/admin_navbar_dropdown.jsx

```diff
@@ -20,9 +20,11 @@ import MenuItemExternalLink from 'components/widgets/menu/menu_items/menu_item_e
 import MenuItemToggleModalRedux from 'components/widgets/menu/menu_items/menu_item_toggle_modal_redux';
 import MenuItemBlockableLink from 'components/widgets/menu/menu_items/menu_item_blockable_link';
 
-export default class AdminNavbarDropdown extends React.Component {
+class AdminNavbarDropdown extends React.Component {
     static propTypes = {
         locale: PropTypes.string.isRequired,
+        siteName: PropTypes.string,
+        intl: intlShape.isRequired,
```

Comment:
We no longer use this approach, we now use the approach that you can see here `components/select_team/components/select_team_item.jsx`. There is a Context provider for intl.